projects/opentrain/israel_railways_pipeline/common/common.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+message routes {
+   int32 id = 1;
+   repeated int32 stop_ids = 2;
+}
+
+
+message samples {
+   int32 id=1;
+   repeated int32 gtfs_stop_id=2;

```

Comment:
Why do we need `repeated` option for these fields?

It looks to me all fields in `samples` message aren't collections. Also in `trips` message.

Reply:
its deleted